modules/forum/src/main/DataForm.scala

```diff
@@ -42,18 +42,18 @@ object DataForm {
   ) {
 
     def looksLikeVenting = List(name, post.text) exists { txt =>
-      mostlyUpperCase(txt) || ventingPattern.matcher(txt).find
+      mostlyUpperCase(txt) || ventingRegex.find(txt)
     }
   }
 
   private def mostlyUpperCase(txt: String) = {
     val extract = txt.take(300)
-    (extract.contains(' ') || extract.size > 5) && {
-      extract.count(_.isUpper) > extract.count(_.isLower) * 2
+    (extract.contains(" ") || extract.size > 5) && {
+      2 * extract.count(_.isUpper) > extract.count(_.isLower)
```

Comment:
functional change intended?

Reply:
there's another very similar line in this patch. that one had a divide

Reply:
Actually maybe I did intend this functional change and saw it as a bug. Have to think about what this means.

Reply:
Yeah I think it was correct before. Will remove